lib/ironic_tls_setup.sh

```diff
@@ -73,4 +92,7 @@ else
     unset IRONIC_INSPECTOR_CACERT_FILE
     unset IRONIC_INSPECTOR_CERT_FILE
     unset IRONIC_INSPECTOR_KEY_FILE
-fi
\ No newline at end of file
+    unset MARIADB_CACERT_FILE
+    unset MARIADB_CERT_FILE
+    unset MARIADB_KEY_FILE
```

Comment:
Would you mind also to update the vars.md with these new vars?

Reply:
I have added that. Thanks